src/normalize/entities.ts

```diff
@@ -15,7 +15,8 @@ const MimePart = new Entity({
 	s: 'size',
 	part: 'part',
 	mid: 'messageId',
-	content: 'content'
+	content: 'content',
+	truncated: 'truncated'
```

Comment:
we don't need to add these if key/value are same

Reply:
also remove part key from line no 16

Reply:
removed these key value pair.